gui/wxpython/image2target/ii2t_manager.py

```diff
@@ -1228,7 +1229,9 @@ def InitMapDisplay(self):
         # initialize column sorter
         self.itemDataMap = self.mapcoordlist
         ncols = self.list.GetColumnCount()
-        ColumnSorterMixin.__init__(self, ncols)  # noqa: PLC2801, C2801
+        # pylint: disable=C2801
+        # ruff: noqa: PLC2801
+        ColumnSorterMixin.__init__(self, ncols)
```

Comment:
This seems for your other PR, where I'm not sure it works as intended for a next line. Teach me and show me if I'm wrong